server/enterprise/metrics/metrics.go

```diff
@@ -209,6 +210,20 @@ type MetricsInterfaceImpl struct {
 	NotificationErrorCounters       *prometheus.CounterVec
 	NotificationNotSentCounters     *prometheus.CounterVec
 	NotificationUnsupportedCounters *prometheus.CounterVec
+
+	ClientTimeToFirstByte        *prometheus.HistogramVec
+	ClientFirstContentfulPaint   *prometheus.HistogramVec
+	ClientLargestContentfulPaint *prometheus.HistogramVec
+	ClientInteractionToNextPaint *prometheus.HistogramVec
+	ClientCumulativeLayoutShift  *prometheus.HistogramVec
+	ClientLongTasks              *prometheus.CounterVec
+	ClientPageLoadDuration       *prometheus.HistogramVec
+	ClientChannelSwitchDuration  *prometheus.HistogramVec
+	ClientTeamSwitchDuration     *prometheus.HistogramVec
+	ClientThreadsLoadDuration    *prometheus.HistogramVec
+	ClientMessagePendingDuration *prometheus.HistogramVec
+	ClientMessagePostedDuration  *prometheus.HistogramVec
+	ClientRHSLoadDuration        *prometheus.HistogramVec
```

Comment:
@hmhealey added the metrics per spec, PTAL.

Reply:
Thanks for adding those. I actually cut `ClientPageLoadDuration` since it's sort of tracked by FCP in a way that's a bit more useful (`page_load` occurs before anything has rendered on screen). The following ones might miss this first release:
- `ClientThreadsLoadDuration
- `ClientMessagePendingDuration`
- `ClientMessagePostedDuration`

Could you maybe remove them for now and we can go back and re-add them later once those are ready? I'm 0/5 on if we want to have metrics around on the server which aren't actually used yet

Reply:
Sure 👍